### DISCUSSION TITLE
Option to check/uncheck all checkboxes, similar to Google Keep

### DISCUSSION THREAD

**Discussion by 20kaushik02** [usememos/memos#4004] [Ideas]
Would be really useful for a list of routine checks, i.e. check one by one, then when you're done uncheck all for the next time. Just a small QoL feature I suppose.

![image](https://github.com/user-attachments/assets/862c2751-f69e-4141-8597-65e50b970b80)

**Comment by Itschotsch** [comment]
I second this. I use this feature often when re-using a checklist. I also think it's extremely easily implemented.
